Shopify/cli
# Show that there are more apps to search

### WHY are these changes introduced?

Fixes https://github.com/Shopify/cli/issues/1147

### WHAT is this pull request doing?

I've added a message to the bottom of autocomplete prompts that have search results with more than 25 items.

![](https://screenshot.click/26-20-5vo3z-c86cj.png)

Unfortunately, because our GraphQL api doesn't expose a `totalCount` property, we can only say if there are more pages or not.

### How to test your changes?

- Run `shopify dev --reset` with an organization that has more than 25 apps
- See the message appear
- The message should disappear after narrowing down the list to less than 25 items

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
